trucks.js

```diff
@@ -501,4 +543,70 @@ var foodTrucks = [
     Facebook: 'https://www.facebook.com/fulltilticecreams',
     Twitter: 'https://twitter.com/fticecream'
   }
-];
\ No newline at end of file
+];
+
+var _ = require('lodash');
+
+function getTrucks() {
+  return foodTrucks;
+}
+
+function getTruck(name) {
+  return foodTrucks.find(function (truck){
+    return truck.name === name;
+  });
+}
+
+/////////////NEW ITEM///////////////////
+function addTruck(newTruck) {
```

Comment:
Note how you're using this method in your truckRoutes.js file - you are expecting it to return a value that you can then send back to the response, but if you look at the implementation of this function, it's missing something.